src/main/kotlin/com/ermolnik/api/users/User.kt

```diff
@@ -0,0 +1,31 @@
+package com.ermolnik.api.users
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class CreateUserRequest(
+    @SerialName("username")
```

Comment:
не избыточно ли SerialName , если филд и нейминг не отличается?

Reply:
I prefer things to be explicit. Another point here is that we will still have to leave one field with `@SerialName` because it differs when serialized but every other one will be without `@SerialName`, which can make the situation confusing and misleading for some people because you will have to know and remember every time how this implicit way of names serialization works.